client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -343,59 +363,89 @@ class ActivityLogTasklist extends Component {
 				</div>
 				{ // Show if plugin update didn't start, is still running or errored,
 				// but hide plugin if it was updated successfully.
-				itemsToUpdate.map( item => (
-					<ActivityLogTaskUpdate
-						key={ item.slug }
-						toUpdate={ item }
-						name={ item.name }
-						slug={ item.slug }
-						version={ item.version }
-						type={ item.type }
-						updateType={
-							'plugin' === item.type
-								? translate( 'Plugin update available' )
-								: translate( 'Theme update available' )
-						}
-						goToPage={ this.goToPage }
-						enqueue={ this.enqueue }
-						dismiss={ this.dismiss }
-						disable={ isItemEnqueued( item.slug, queued ) }
-					/>
-				) ) }
+				itemsToUpdate.map( item => {
+					let updateType = translate( 'Plugin update available' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Plugin update available', 'Plugin updates available' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/activity-log/core-updates)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).